hax/hax/server.py

```diff
@@ -89,6 +89,21 @@ def hctl_stat(request):
     return json_response(text=result)
 
 
+def hctl_fetch_fids(request):
```

Comment:
request parameter is unused, also reported by codacy. Please remove the same.

Reply:
@mssawant This function is request handler and it needs to accept Request instance as input argument
https://docs.aiohttp.org/en/stable/web_quickstart.html#handler

Reply:
Okay, let's document this then,
```
This function implements the http fetch-fids request and thus, takes request a parameter.
The respective result is obtained by executing corresponding `hctl-fetch-fids` script.
```